.github/workflows/cut-rc.yml

```diff
@@ -446,13 +446,13 @@ jobs:
           tar xf oc.tar
       - name: Patch central
         run: |
-          ./oc -n stackrox set image deploy/central central=docker.io/stackrox/main:$TAG
+          ./oc -n stackrox set image deploy/central central=quay.io/stackrox-io/main:$TAG
```

Comment:
I believe all mentions of `quay.io/stackrox-io` should be replaced with `quay.io/rhacs-eng` in this file. The reason is that RC is concerned with commercially-branded images and these are in `/rhacs-eng`.